packages/simcore-sdk/src/simcore_sdk/node_ports_v2/nodeports_v2.py

```diff
@@ -127,5 +130,13 @@ async def set_multiple(self, port_values: Dict[str, ItemConcreteValue]) -> None:
                 # if this fails it will raise another exception
                 tasks.append(self.internal_inputs[port_key]._set(value))
 
-        await logged_gather(*tasks)
+        results = await logged_gather(*tasks)
         await self.save_to_db_cb(self)
+
+        # groups all ValidationErrors pre-pending 'port_key' to loc and raises ValidationError
+        if errors := [
+            flatten_errors(r, self.__config__, loc=(f"{port_key}",))
+            for port_key, r in zip(port_values.keys(), results)
+            if isinstance(r, ValidationError)
+        ]:
+            raise ValidationError(errors, model=self)
```

Comment:
Great, I am sure this will be of much help!

Would you mind testing if `jupyter-math:2.0.5` is still working after this change?
Could you start such a service and drop a file in one of the output ports?

Please make sure no logs are displayed in the logger inside the of osparc.